external-crates/move/crates/move-compiler/tests/move_2024/migration/let_mut_migrate.migration.exp

```diff
@@ -2,7 +2,7 @@
 +++ tests/move_2024/migration/let_mut_migrate.move
 @@ -3,1 +3,1 @@
 -    public fun t(x: u64, yip: u64, s: S): u64  {
-+    public fun t(mut x: u64, mut yip: u64, s: S): u64  {
++    public fun t(mut x: u64, yip: u64, s: S): u64  {
```

Comment:
While I appreciate the semantic correctness here, it is unnerving. :)

Reply:
Wait what? This should hav emut still? Maybe an issue of unused bindings again?